homework1/task3_parts_by_coords.py

```diff
@@ -0,0 +1,24 @@
+"""Определение номера четверти плоскости по координаты точки (X и Y), причём X ≠ 0 и Y ≠ 0"""
+
+x = 0
+y = 0
+
+#цикл для проверки того, что пользователь не ввел точку (0,0)
+while x == 0 and y == 0:
+    #стараюсь соблюдать PEP8 и ограничивать строку 80 символами
+    #поэтому в следующей строке добавился print() перед input()
+    print('Введите координаты точки (X ≠ 0 и Y ≠ 0) через пробел:',end=" ")
+    x,y = map(float,(input().split(sep=' ')))
```

Comment:
```suggestion
    x,y = map(float, input().split())
```
Если хотите убрать пробелы, то лучше не передавать методу `split` никаких аргументов.

Reply:
Спасибо. Зафиксировал у себя в голове

Reply:
> Если хотите убрать пробелы, то лучше не передавать методу `split` никаких аргументов.

Спасибо. Зафиксировал у себя в голове